includes/services/check_dns.inc.php

```diff
@@ -9,7 +9,11 @@
 if ($service['service_ip']) {
     $resolver = $service['service_ip'];
 } else {
-    $resolver = $service['hostname'];
+    if ($service['overwrite_ip']) {
```

Comment:
why not using elseif to avoid leveled if statements?

Reply:
That is much cleaner. Thanks!